KabakiAntony/todos
# Finishes building user verfication route.

#### What does this PR do?
Finishes user verification route
#### Description of Task to be completed?
Finishes building a user email verification route, that is when a user signs up with the application they cannot use it before 
they verify their email account.
#### How should this be manually tested?
By getting the token in the signup success email, copy the jwt token part, and then got to the /users/verify route on postman and add auth_token under headers and enter the copied token as the value
#### Any background context you want to provide?
Users cannot start using the system if they have not verified their email accounts.
